webapp/portlet/src/main/webapp/vue-apps/attach-image/components/form/ImageInputMultiUpload.vue

```diff
@@ -44,6 +44,18 @@ export default {
   },
   created() {
     document.addEventListener('open-file-explorer', this.triggerFileClickEvent);
+    document.addEventListener('paste', event => {
+      if (event?.clipboardData?.files) {
+        const files = event.clipboardData.files;
+        const imageItems = [].slice.call(files).filter((item) => {
+          return item.type.indexOf('image') !== -1;
+        });
+        if (!imageItems.length) {
+          return;
+        }
+        this.uploadFiles(imageItems);
+      }
+    }); 
```

Comment:
1. Add the method inside a method to be able to delete it from `beforeDestroy`
2. Add `document.removeEventListener` in `beforeDestroy`
3. Use `clipboardData?.files || event?.clipboardData?.items` instead of `clipboardData?.files` only to include copying images from Paint Softwares...

```suggestion
    document.addEventListener('paste', this.handlePasteFiles); 
  },
  beforeDestroy() {
     ...
     document.removeEventListener('paste', this.handlePasteFiles); 
  },
  methods: {
    handlePasteFiles(event) {
      const files = event?.clipboardData?.files || event?.clipboardData?.items;
      if (files?.length) {
        const files = event.clipboardData.files;
        const imageItems = files.filter(file => file?.type?.includes('image/'));
        if (imageItems.length) {
          this.uploadFiles(imageItems);
        }
      }
    },
    ...
```

Reply:
done